operations/workspace/deployment/pkg/orchestrate/deploy.go

```diff
@@ -35,11 +35,15 @@ func Deploy(context *common.Context, clusters []*common.WorkspaceCluster) error
 	return nil
 }
 
-// TODO(prs): Add implementation once we have scripts in ops repo tested and
-// install step implemented
 func installGitpod(context *common.Context, cluster *common.WorkspaceCluster) error {
-	log.Log.Infof("received request to install gitpod on cluster: %s cannot be processed. Implementation pending", cluster.Name)
-	return nil
+	err := step.InstallGitpod(context, cluster)
```

Comment:
Do line 39 and 40 want to swapped around?

Reply:
good catch

Reply:
Fixed